zerver/tests/test_bugdown.py

```diff
@@ -169,10 +172,13 @@ def bugdown_convert(content: str) -> str:
     message = cast(Message, FakeMessage())
     message.content = content
     message.id = 999
+    empty_automaton = ahocorasick.Automaton()
+    empty_automaton.make_automaton()
     return bugdown.convert(
         content=content,
         message_realm=get_realm('zulip'),
         message=message,
+        realm_alert_words_automaton=empty_automaton
```

Comment:
Same comment on `None` here.

Reply:
True, `bugdown.convert` can take in a `None` and work just fine.